data/json/items/armor/bandolier.json

```diff
@@ -138,5 +138,32 @@
         "draw_cost": 20
     },
     "flags": [ "WATER_FRIENDLY", "BELTED", "OVERSIZE" ]
-  }
+  },
+  {
+    "id": "grenadebandolier",
+    "type": "ARMOR",
+    "name": "large grenade pouch",
+    "name_plural": "large grenade pouches",
+    "description": "A pouch for holding up to four full-sized grenades of various types.",
+    "weight": 600,
+    "volume": 5,
+    "price": 10000,
```

Comment:
10000 for a simple grenade pouch? More than a EMP grenade (6000), or even the ordinary grenade (1500)?